README.md

```diff
@@ -30,15 +30,21 @@ Consumers of the domain data should only access data through the product views.
 across releases of the MDS.
 
 ## Configuration
+Source datasets and destination datasets are controled by the values in [dataform.json](/dataform.json) file.
+
+### Source datasets
+Variables:
+* `ga4_export_project` - GA4 project
+* `ga4_export_datase` - GA4 dataset
+* `ga4_export_table_suffix` - typically `_*`
+* `ga4_incremental_processing_days_back` - number of days to look back when processing incremental data. Default is 3.
+    
+* `ads_export_data` - an array of datasets where Ads data is exported. See the file for the exact syntax.
+* `ads_metrics_lookback_days` - number of days to calculate Ad metrics in the dashboard reports. Default is 365.
```

Comment:
Apologies, I should have given the details on this addition. This variable is for determining number of days to query and load from the Ads DTS landing tables to the fact tables for Ads Performance and Ads Conversions.

Can we patch this change or should we approve and make another?

Reply:
Updated the description. Resulted in another commit.